oss-internship-2020/pffft/README.md

```diff
@@ -0,0 +1,92 @@
+# Sandboxing PFFFT library
+
+Build System: CMake  
+OS: Linux
+
+### Check out the PFFFT library & CMake set up
+`git checkout -b master`
+
+`git submodule update --init --recursive`
+
+`mkdir -p build && cd build`
+
+`cmake .. -G Ninja -DPFFFT_ROOT_DIR=$PWD`
+
+`ninja`
+
```

Comment:
You can mark blocks of code in Markdown using ` ``` `:

    ```bash
    git submodule update --init --recursive
    
    mkdir -p build && cd build
    cmake .. -G Ninja -DPFFFT_ROOT_DIR=$PWD`
    ninja
    ```